### ISSUE TITLE
Complete bibliography item for reference 26.

### ISSUE THREAD

**Issue by tmcphillips** [whole-tale/reproducibility-components#5]
Reviewer comment R2.20:

> R2.20  It looks like reference [26] is missing some information.

The reference is missing authors and title.

**Comment by tmcphillips** [comment]
The bibtex entry for [26] is:
```
@proceedings{DBLP:conf/eScience/2018,
  title     = {14th {IEEE} International Conference on e-Science, e-Science 2018,
               Amsterdam, The Netherlands, October 29 - November 1, 2018},
  publisher = {{IEEE} Computer Society},
  key = {RO2018}, 
  year      = {2018},
  url       = {http://ieeexplore.ieee.org/xpl/mostRecentIssue.jsp?punumber=8588285},
  isbn      = {978-1-5386-9156-4},
  timestamp = {Fri, 04 Jan 2019 13:57:07 +0100},
  biburl    = {https://dblp.org/rec/bib/conf/eScience/2018},
  bibsource = {dblp computer science bibliography, https://dblp.org}
```

This appears to refer to the eScience 2018 proceedings as a whole.  @ludaesch Was this reference meant to refer to a specific article?

**Comment by ludaesch** [comment]
Hmm.. Yes, it's a citation to the proceedings and thus probably a glitch. I think what happened is that I wanted to \cite{mecum2018preservingc} and that paper is part of the proceedings "DBLP:conf/eScience/2018". (This might have been a Bibtex @crossref link that somehow went bad). 

Possible fixes:
(1)  simply remove the DBLP:conf/eScience/2018 entry
(2) make a proper "crossref" that works.

I'll take care of it.

@inproceedings{mecum2018preservingc,
  title = {Preserving {{Reproducibility}}: {{Provenance}} and {{Executable Containers}} in {{DataONE Data Packages}}},
  shorttitle = {Preserving {{Reproducibility}}},
  abstract = {Many data packaging standards are available to researchers and data repository operators and the choice to use an existing standard or create a new one is challenging. We introduce the DataONE Data Package standard which is based on the existing OAI-ORE Resource Map standard. We describe the functionality Data Package provides, implementation considerations, compare it to existing standards, and discuss future extensions to the standard including the ability to describe execution environments via WholeTale "Tales"" and alternate serialization formats.},
  booktitle = {2018 {{IEEE}} 14th {{International Conference}} on E-{{Science}} (e-{{Science}})},
  doi = {10.1109/eScience.2018.00019},
  author = {Mecum, B. and Jones, M. B. and Vieglais, D. and Willis, C.},
  month = oct,
  year = {2018},

**Comment by ludaesch** [comment]
I tried both versions (1) removing the citation, and (2) adding the crossref.
Turns out that (2) creates two refs as expected, but drops the page number, so uses more space and is less informative. I think we should go with (1) instead, i.e., replace
"... supporting transparency~\cite{mecum2018preservingc,DBLP:conf/eScience/2018}"
with
"... supporting transparency~\cite{mecum2018preservingc}" 

That will be the better fix.

FYI: I tried to push the change but might not have permission to do so.

Bertram
